catalog-entities/marketplace/plugins/all.yaml

```diff
@@ -27,11 +27,13 @@ spec:
     - ./github-actions.yaml
     - ./github-catalog-integration.yaml
     - ./github-insights.yaml
+    - ./github-org-catalog-integration.yaml
```

Comment:
Sorry, but  file has name`github-org-catalog-integrations.yaml` - with "s". You need or rename file or fix the current path.